qubes/devices.py

```diff
@@ -63,10 +63,13 @@
 
 import qubes.exc
 import qubes.utils
-from qubes.device_protocol import (Device, DeviceInfo, UnknownDevice,
-                                   DeviceAssignment)
+from qubes.device_protocol import (Port, DeviceInfo, UnknownDevice,
+                                   DeviceAssignment, VirtualDevice,
+                                   AssignmentMode)
 
 
+DEVICE_DENY_LIST = "/etc/qubes/device-deny.list"
```

Comment:
As discussed on the summit, this wants to be a directory (`/etc/qubes/device-deny.list.d`?)